lib/yao/resources/restfully_accessible.rb

```diff
@@ -150,5 +156,10 @@ def return_resource(d)
     def return_resources(arr)
       arr.map{|d| return_resource(d) }
     end
+
+    def is_uuid?(str)
+      /^[\da-f]{32}$/ === str or
```

Comment:
I have no idea too.Short UUID is almost not used (used in keystone only. maybe). 
So, I will delete it.